VDF.GUI/ViewModels/MainWindowVM_Settings.cs

```diff
@@ -110,7 +110,7 @@ public partial class MainWindowVM : ReactiveObject {
 			if (result == null || result.Count == 0) return;
 			foreach (var item in result) {
 				if (!SettingsFile.Instance.Includes.Contains(item))
```

Comment:
Also here in the if: Includes <-> Blacklists

Reply:
Fixed, thanks!